src/components/ontologies/Ontologies.module.css

```diff
@@ -26,4 +26,23 @@
     display: flex;
     justify-content: flex-start;
     width: 100%;
-}
\ No newline at end of file
+}
+
+.newCarLoader {
```

Comment:
naming breaks with css naming conventions of using hypen as separator. Please switch from camel-case to hyphen-separation